src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,18 +24,32 @@ const (
     // Always begin a "tag" with a dash (as per http://semver.org)
     tag   = "-beta"
 
-    semVerFormat = "%s.%s.%s%s"
     userAgentFormat = "Azure-SDK-For-Go/%s arm-%s/%s"
 )
 
+var userAgent string
```

Comment:
I think this looks more similar to other declarations, but still it is just a nit
``` go
var (
 userAgent string
 version string
)
```